platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/LocalRecipeDaoImpl.java

```diff
@@ -65,13 +52,12 @@
     private final ReadWriteLock              lock;
     private final LocalStorage               recipeStorage;
 
-    @javax.inject.Inject
-    @Named("local.storage.path")
-    private String pathToStorage;
-
     @Inject
     public LocalRecipeDaoImpl(LocalStorageFactory storageFactory) throws IOException {
-        this.recipeStorage = storageFactory.create("recipes.json");
+        Map<Class<?>, Object> adapters = new HashMap<>();
+        adapters.put(Permissions.class, new PermissionsDeserializer());
+        adapters.put(Group.class, new GroupDeserializer());
+        this.recipeStorage = storageFactory.create("recipes.json", adapters);
```

Comment:
http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/collect/ImmutableMap.html#of(K,%20V)